saleor/core/utils/random_data.py

```diff
@@ -533,3 +538,16 @@ def add_address_to_admin(email):
     address = create_address()
     user = User.objects.get(email=email)
     store_user_address(user, address, True, True)
+
+
+def create_fake_collection():
+    collection = get_or_create_collection(name='%s collection' % fake.word())
+    for product in Product.objects.all().order_by('?')[:4]:
+        collection.products.add(product)
```

Comment:
I'd assign above queryset to a variable with the meaningful name, and bulk add those products to the collection